mkdocs.yml

```diff
@@ -11,6 +11,7 @@ extra_javascript:
 
 extra_css:
   - static/css/asciinema-player.css
+  - stylesheets/extra.css
```

Comment:
Maybe we could move `extra.css` to `static/css/extra.css` and remove the `stylesheets` directory?